ansible/roles/php/tasks/main_env.yml

```diff
@@ -51,7 +51,8 @@
   set_fact:
     php__version: '{{ ansible_local.php.version|d(php__register_version.stdout) }}'
     php__long_version: '{{ ansible_local.php.long_version
-                           if (ansible_local.php.long_version|d() != "(none)")
+                           if (ansible_local|d() and ansible_local.php|d() and
+                               ansible_local.php.long_version|d() != "(none)")
```

Comment:
Does the Ansible 2.8 shortcut not work here?

Reply:
Apparently, you can see the error [in the CI pipeline test](https://gitlab.com/debops/debops/-/jobs/601197359). I'm not sure what caused this, it also shows up with the `debug` module, so it's probably a quirk of Jinja.

Reply:
Wired. Also does not make sense to me. Ok then.